docs/README.md

```diff
@@ -9,6 +9,38 @@ See [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.htm
 
 ## Rules for writing documents ##
 
+### Markup Rules
+
+* put two single quotes around an option name like ``` ``--help`` ```.
+
+* put two single quotes with colon around field name like ``` ``field:`` ```.
+
+* put double quotes aground an example of command line, option usage like
+  "--langdef=MyLang", referring a section, and a file path.
+
+* use two asterisks for newly introduced conceptual `**word**` and reserved
+  words. The reserved words are
+
+  * "default" in --langmap
+
+* use one asterisk for option parameter like `*LANG*` in --kind-<LANG> option.
```

Comment:
I wonder which one we should use `*MATAVAR*`, `<METAVAR>`, or `*<METAVAR>*`.
In ``--help`` output, we have used  `<METAVAR>` because u-ctags did so.
Just using the uppercase string `METAVAR` is not enough in the help message. Let's think `C`. 
People may wonder whether it represents a name of a parser or a metavariable.
We cannot use a style like bold or italic in the help message.
So using `<` and `>` is understandable. 

On the man page, we can use a style for the purpose. So I wonder how we should do.
I'm familiar with `<...>` notation, so I would like to use `<METAVAR>` or `*<METAVAR>*`.

As far as I know, currently `<` and `>` are used only for `LANG`. I would like to use the notation for
other places like `<KIND>`.

This is just a comment.

Reply:
It seems that we should use `*<METAVAR>*` in the man pages.

Reply:
Yes.  As you already found, on 7667692 I moved the contents from `man/README` to `docs/README.md` **as is**.
`*<METAVAR>*` in e37d287 is my proposal.